src/js/views/CovidDashboardView.js

```diff
@@ -0,0 +1,881 @@
+    this.tableButtonPrev = elementFactory('button', { class: 'arrow-button' }, leftArrow);
+    this.tableButtonNext = elementFactory('button', { class: 'arrow-button' }, rightArrow);
+
+    const searchLabel = elementFactory('label', { class: 'search-label', for: 'searchInput' }, 'Search country: ');
+    this.tableFilterInput = elementFactory('input', { class: 'country-search', id: 'searchInput' }, 'search country');
+
+    const searchGroup = elementFactory('div', { class: 'search-group' }, searchLabel, this.tableFilterInput);
+
+    this.tableHeader = elementFactory('div', { class: 'table-header' }, properties[0].header);
+
+    const tableControl = elementFactory('div', { class: 'table-control' }, this.tableButtonPrev, this.tableHeader, this.tableButtonNext);
+
+    this.periodInput = elementFactory('input', { type: 'checkbox', class: 'period-check', checked: true }, '');
+    const periodSlider = elementFactory('span', { class: 'slider round', 'data-on': 'Total', 'data-off': 'Last day' }, '');
+    this.togglePeriodButton = elementFactory('label', { class: 'switch' }, this.periodInput, periodSlider);
+    const periodSwitchHeader = elementFactory('span', { class: 'control-header' }, 'Period: ', this.togglePeriodButton);
+
+    this.populationInput = elementFactory('input', { type: 'checkbox', class: 'population-check', checked: true }, '');
+    const populationSlider = elementFactory('span', { class: 'slider round', 'data-on': 'Total', 'data-off': 'Per 100k' }, '');
+    this.togglePopulationButton = elementFactory('label', { class: 'switch' }, this.populationInput, populationSlider);
+    const populationSwitchHeader = elementFactory('span', { class: 'control-header' }, 'Population: ', this.togglePopulationButton);
+
+    const resizeImg = elementFactory('img', { class: 'resize-img', src: 'assets/img/enlarge.svg', alt: 'resize-img' }, '');
+    this.countryTableResize = elementFactory('button', { class: 'resize-button' }, resizeImg);
+
+    const controlGroup = elementFactory('div', { class: 'control-group' }, periodSwitchHeader, populationSwitchHeader);
+    this.table = elementFactory('div', { class: 'country-table data-load-box' }, this.loadData);
+
+    const header = elementFactory('div', { class: 'country-container-header' }, 'Cases by Counry/Region/Province');
+    this.container = elementFactory('div', { class: 'country-table-container' }, this.countryTableResize, header, controlGroup, searchGroup, tableControl, this.table);
+    this.tableMapContainer = elementFactory('div', { class: 'table-map-wrapper' }, this.container);
+
+    this.simpleKeyboard = elementFactory('div', { class: 'simple-keyboard' }, '');
+    document.body.appendChild(this.simpleKeyboard);
+
+    this.keyboard = new Keyboard({
+      onChange: (input) => {
+        this.tableFilterInput.value = input;
+        this.filterTable(input);
+      },
+      onKeyPress: (button) => {
+        if (button === "{enter}") {
```

Comment:
please be consistent with quotes for strings use either single-quotes or double-quotes across all your files